server/src/test/java/org/elasticsearch/cluster/metadata/MetaDataIndexStateServiceTests.java

```diff
@@ -191,14 +240,6 @@ public void testAddIndexClosedBlocks() {
             ClusterState state = addOpenedIndex("index-1", randomIntBetween(1, 3), randomIntBetween(0, 3), initialState);
             state = addOpenedIndex("index-2", randomIntBetween(1, 3), randomIntBetween(0, 3), state);
             state = addOpenedIndex("index-3", randomIntBetween(1, 3), randomIntBetween(0, 3), state);
-            final boolean mixedVersions = randomBoolean();
-            if (mixedVersions) {
-                state = ClusterState.builder(state)
-                    .nodes(DiscoveryNodes.builder(state.nodes())
-                        .add(new DiscoveryNode("old_node", buildNewFakeTransportAddress(), emptyMap(),
-                             new HashSet<>(Arrays.asList(DiscoveryNode.Role.values())), Version.V_6_0_0)))
```

Comment:
Would it maybe make sense to still verify this works in the mixed cluster case (against 7.x)?

Reply:
Right, I added a note in https://github.com/elastic/elasticsearch/issues/33888 so that I won't forgot to keep this test in 7.x